.github/workflows/ci.yaml

```diff
@@ -50,10 +50,25 @@ jobs:
           args: ${{ env.CARGO_ARGS }} --no-default-features
 
   exotic_targets:
-    name: Ensure compilation on exotic targets
+    name: Ensure compilation on various targets
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          target: i686-unknown-linux-gnu
+
+      - uses: Swatinem/rust-cache@v1
```

Comment:
I see the same action being used further down, shouldn't only one exist? (Preferably the one on this line since we need the cache to be present before we run `check`).

Looking[ at the run ](https://github.com/RustPython/RustPython/blob/main/.github/workflows/ci.yaml#L40)for `rust_tests` it _seems_ like one time should be enough.

Reply:
uh, this is weird. I read the project's document: https://github.com/Swatinem/rust-cache#example-usage
So I thought they will cache proper items by `action-rs/toolchain`. But the cache key prefix is `v0-rust-exotic_targets-1.55.0-x86_64-unknown-linux-gnu`. I think cache is not working for cross targets. I will remove them.

Reply:
https://github.com/Swatinem/rust-cache/issues/33

Reply:
Gotcha. It looks fine to me for now. We can add the caching at a later point.